bot/kodiak/queue.py

```diff
@@ -334,4 +334,10 @@ def get_webhook_queue_name(event: WebhookEvent) -> str:
     return f"webhook:{event.installation_id}"
 
 
-redis_webhook_queue = RedisWebhookQueue()
+async def enqueue_raw_webhook(
+    *, redis: asyncio_redis.Connection, event_name: str, event: dict[str, object]
+) -> None:
+    logger.info("enqueuing github webhook event")
+
+    # TODO(sbdchd): how do we want to format the queue messages?
+    await redis.rpush(event)
```

Comment:
@chdsbd how do we want to format the incoming raw events from GitHub?

Reply:
Maybe something like this. I think we need to record the event name, which is passed in via an HTTP header. We can probably dump the payload body as a string from JSON
```py
class RawEvent:
    name: str
    payload: str
```